services/cd-service/pkg/service/service.go

```diff
@@ -72,8 +72,13 @@ func (s *Service) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (s *Service) ServeHTTPHealth(w http.ResponseWriter, r *http.Request) {
-	w.WriteHeader(200)
-	fmt.Fprintf(w, "ok\n")
+	if err := s.Repository.State().ValidateConfigs(); err != nil {
```

Comment:
This seems a bit expensive to do in every health check

Reply:
I agree. Let's remove this from the health check.